graylog2-web-interface/src/views/components/Query.jsx

```diff
@@ -56,7 +58,27 @@ const _renderWidgetGrid = (widgetDefs, widgetMapping, results, positions, queryI
   );
 };
 
+const _emptyDashboardInfo = () => {
```

Comment:
I used the underscore for the name, because all other "sub" functions do, whats the preferred way? :)

Reply:
Answer: undersore for private functions, in my case an own component makes more sense